LibSWBF2/Wrappers/Terrain.cpp

```diff
@@ -113,6 +113,80 @@ namespace LibSWBF2::Wrappers
 		return true;
 	}
 
+	Terrain::Terrain()
+	{
+		p_HeightMap = nullptr;
+		p_BlendMap = nullptr;
+	}
+
+	Terrain::~Terrain()
+	{
+		delete p_HeightMap;
+		delete p_BlendMap;
+	}
+
```

Comment:
Management of blend/heightmap mem moved to wrapper as lazily initialized buffers.  Chose not to make them ```List```s since they are populated in a non-sequential manner.